parquet-protobuf/src/main/java/org/apache/parquet/proto/ProtoWriteSupport.java

```diff
@@ -318,7 +318,7 @@ final void writeRawValue(Object value) {
     @Override
     final void writeRawValue(Object value) {
       ByteString byteString = (ByteString) value;
-      Binary binary = Binary.fromByteArray(byteString.toByteArray());
+      Binary binary = Binary.fromUnmodifiedByteArray(byteString.toByteArray());
```

Comment:
Is ByteString immutable? If it isn't, then I think this should be `fromReusedByteArray`.